blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,284 @@
+---
+title: We are not giving up on the cloud!
+description: Announcing our seed funding round for Wing Cloud
+authors: 
+  - eladb
+tags: [wing cloud, funding, seed, press release]
+hide_table_of_contents: true
+---
+
+All comedowns are painful, and it seems like our come down from the cloud's [hype
+curve](https://en.wikipedia.org/wiki/Gartner_hype_cycle) is no different. When
+[@dhh](https://twitter.com/dhh) writes about ["Why we're leaving the
+cloud"](https://world.hey.com/dhh/why-we-re-leaving-the-cloud-654b47e0) and Prime Video are
+[mourning the unexpected costs of
+serverless](https://www.primevideotech.com/video-streaming/scaling-up-the-prime-video-audio-video-monitoring-service-and-reducing-costs-by-90),
+it's not easy to read, especially for a cloud believer like myself.
+
+And even if companies are not stocking up on server racks, there's a strong sentiment of compromise.
+The complexity is overwhelming and teams are struggling to set up reasonable development
+environments, test their systems or deploy them across multiple public providers or private clouds. 
+
+I don't think I've ever talked to a platform team that feels like they got it right, or to a DevOps
+engineer that didn't feel they are a bottleneck. I hear frustration and apologies "I know this is
+not ideal, but we've got to give developers a way to develop". When I talk to enterprises, many of
+them are in “analysis paralysis”. Which cloud provider do we sell our souls to? Are we a serverless
```

Comment:
nit: In a lot of writing I've seen, periods tend to go inside quotes at an end of a sentence

Reply:
done

Reply:
There's a whole philosophy behind it (UK/US I think).